LeaderboardBackend/Controllers/LeaderboardsController.cs

```diff
@@ -9,14 +9,9 @@
 
 namespace LeaderboardBackend.Controllers;
 
-public class LeaderboardsController : ApiController
+public class LeaderboardsController(ILeaderboardService leaderboardService) : ApiController
 {
-    private readonly ILeaderboardService _leaderboardService;
-
-    public LeaderboardsController(ILeaderboardService leaderboardService)
-    {
-        _leaderboardService = leaderboardService;
-    }
+    private readonly ILeaderboardService _leaderboardService = leaderboardService;
```

Comment:
You don't need to store the parameter in a field.

Reply:
Eh

Guess I forgot to remove this